projects/elife.yaml

```diff
@@ -333,7 +333,9 @@ api-gateway:
             - 443
             # - 8000: # don't expose this to public. Kong uses this to proxy requests
             # - 8001: # don't ever expose to public. Kong uses this for API admin
-
+        fastly:
+            subdomains:
+                - "{instance}--fastly-gateway"
```

Comment:
Automatically points to `full_hostname`. More than one should be supported, also it's useful to have `subdomains-without-dns` too which will allow migrations (where both CloudFront and Fastly CDNs are in place and the DNS is switched from one to another during the transition period).